src/css/custom.css

```
@@ -5,7 +5,20 @@
  */
 
 /* You can override the default Infima variables here. */
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600;700&display=swap');
+
+@font-face {
+  font-family: 'Atyp'; /* You can name it anything */
+  src: url('/src/fonts/AtypDisplay-Bold.woff2') format('woff2'),
+       url('/src/fonts/AtypDisplay-Medium.woff2') format('woff2'),
+       url('/src/fonts/AtypDisplay-Regular.woff2') format('woff2'),
+       url('/src/fonts/AtypDisplay-Semibold.woff2') format('woff2');
```

Consider that once the library outputs to the build directory, ultimately, what the user will see, the "src" directory won't exist. Does the build process replace this path and place the static files there? If so, what's the final directory path?

For example, I'd expect it to be in `assets/fonts`, as for `[ROOT]/build/assets/fonts`, or in the static directory for it to be available in the `build`.

```suggestion
  src: url('/assets/fonts/AtypDisplay-Bold.woff2') format('woff2'),
       url('/assets/fonts/AtypDisplay-Medium.woff2') format('woff2'),
       url('/assets/fonts/AtypDisplay-Regular.woff2') format('woff2'),
       url('/assets/fonts/AtypDisplay-Semibold.woff2') format('woff2');
```